src/Core/Pages/Accounts/EnvironmentPageViewModel.cs

```diff
@@ -73,7 +73,7 @@ public async Task SubmitAsync()
             IconsUrl = resUrls.Icons;
             NotificationsUrls = resUrls.Notifications;
 
-            SubmitSuccessAction?.Invoke();
+            await SubmitSuccessTask();
```

Comment:
⚠️ Just in case check `SubmitSuccessTask` is not `null` before invoking it:
```suggestion
            if (SubmitSuccessTask != null)
            {
                await SubmitSuccessTask();
            }
```